src/string.rs

```diff
@@ -66,7 +66,17 @@ pub fn rewrite_string<'a>(s: &str, fmt: &StringFormat<'a>) -> Option<String> {
                 // anywhere that doesn't break an escape sequence.
                 cur_end = cur_start + max_chars;
                 while graphemes[cur_end - 1] == "\\" {
+                //look for punctuation to break on
+                while graphemes[cur_end - 1] != ":" && graphemes[cur_end - 1] != "," && graphemes[cur_end - 1] != ";" && graphemes[cur_end - 1] != "." {
```

Comment:
This is causing the compile error, you need to remove the previous `while...` line at line 68